mopidy_ytmusic/library.py

```diff
@@ -901,7 +901,9 @@ def albumToTracks(self, album, bId):
                 ):
                     songartists = artists
                 else:
-                    songartists = [Artist(name=song["artists"])]
+                    songartists = []
+                    for artist in song["artists"]:

```

Comment:
Do we know if this is always a list here?

I was going to submit a pull request which checked if it's a list and falls back to the existing else, like this:

```python
                elif type(song["artists"]) is list:
                    # handle case where we have a list of dictionaries like:
                    #   [{'name':..., 'id':...}, ]
                    name = ",".join(sa["name"] for sa in song["artists"])
                    songartists = [Artist(name=name)]
                else:
                    songartists = [Artist(name=song["artists"])]
```

but I am new to using this project, so I'm not sure if we need the extra check or not.